src/main/java/org/spectrumauctions/sats/core/model/lsvm/LSVMBidder.java

```diff
@@ -153,11 +155,7 @@ public double calculateFactor(int size) {
         for (LSVMLicense license : world.getLicenses()) {
```

Comment:
Could you adapt the comment for `calculateFactor(int size)` (https://github.com/spectrumauctions/sats/pull/60/files#diff-9e979b6c8f67aff9704a30c8190cc31aR141) according to the new implementation?

Reply:
Sure. I added some more comments that hopefully help to clarify the situation.